api/batch/src/main/java/it/gov/pagopa/rtd/transaction_filter/batch/step/tasklet/TransactionSenderRestTasklet.java

```diff
@@ -20,7 +20,7 @@
 public class TransactionSenderRestTasklet implements Tasklet, InitializingBean {
 
     private static final int RETRY_MAX_ATTEMPTS = 3;
-    private static final int INITIAL_DELAY_SECS = 2;
+    private int initialDelayInSeconds = 2;
```

Comment:
@GiovanniMancini do we want to parametrize this field? As is the upload will be repeated after 2, 4, 16 seconds.